CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Current (in progress)
 
 - Adds latest `sentry-sdk[flask]` as an install dependency, and update Sentry logic to be able to send environment, app version and profiling/performance info [#3086](https://github.com/opendatateam/udata/pull/3086)
+- Fix the version of udata-fixtures used by `udata import-fixtures`
```

Comment:
I think you can add the link to this PR for context :)
```suggestion
- Fix the version of udata-fixtures used by `udata import-fixtures` [#3114](https://github.com/opendatateam/udata/pull/3114)
```

Reply:
oh, right, sure, i'll do that straight away and then merge 🙏